upload/garrysmod/gamemodes/clockwork/framework/sh_kernel.lua

```diff
@@ -727,6 +727,10 @@ function Clockwork:HandlePlayerSwimming(player)
 		end;
 		
 		player.m_bInSwim = true;
+		
+		if (player.m_bInSwim) then
```

Comment:
Surely as this variable is defined two lines above, it'd be redundant to do a check here for if the value is true?